src/User.php

```diff
@@ -4854,9 +4854,9 @@ public function showItems($tech)
                 'glpi_groups_users.groups_id',
```

Comment:
May be easier to read.
```suggestion
                'glpi_groups.id',
```